src/neo4j_graphrag/__init__.py

```diff
@@ -12,3 +12,10 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+
+import warnings
```

Comment:
To be removed, now users will be importing from neo4j-graphrag, so all good.

Reply:
https://github.com/neo4j/neo4j-genai-python/pull/130/commits/76696284a904d7dfc6a1d80dd44926f9ea73a31f